core/src/main/java/org/frankframework/util/Dir2Map.java

```diff
@@ -93,13 +91,14 @@ private Map<String, Object> FileInfo(File file) {
 	}
 
 	private Map<String, Object> FileInfo(File file, String displayName) {
-		Map<String, Object> fileInfo = new HashMap<>(5);
+		Map<String, Object> fileInfo = new HashMap<>(6);
 
 		fileInfo.put("name", displayName);
 		fileInfo.put("path", normalizePath(file));
 		fileInfo.put("lastModified", file.lastModified());
 		fileInfo.put("type", file.isDirectory() ? "directory" : "file");
-		fileInfo.put("size", (sizeFormat) ? Misc.toFileSize(file.length(), true) : file.length());
+		fileInfo.put("size", file.length());
+		fileInfo.put("sizeDisplay", Misc.toFileSize(file.length(), true));
```

Comment:
Instead of fetching file size twice it is slightly faster to fetch it once and put it into a local variable.